showcase/app/components/page-components/copy/button/code-fragments/with-generic-dialog-content.gts

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import type { TemplateOnlyComponent } from '@ember/component/template-only';
+
+import {
+  HdsCopyButton,
+  HdsFormTextInputBase,
+} from '@hashicorp/design-system-components/components';
+
+import ShwDivider from 'showcase/components/shw/divider';
+
+export interface CodeFragmentWithGenericDialogContentSignature {
+  Element: HTMLDivElement;
+}
+
+const CodeFragmentWithGenericDialogContent: TemplateOnlyComponent = <template>
+  <div class="shw-component-copy-button-demo-container">
```

Comment:
[nit]: to avoid potantial confusion, I suggest always including splattributes. There are couple of very useful rules that can be customized and catch this cases
* https://github.com/ember-template-lint/ember-template-lint/blob/main/docs/rule/require-splattributes.md
* https://github.com/ember-template-lint/ember-template-lint/blob/main/docs/rule/no-nested-splattributes.md